osu.Game/Rulesets/Objects/Legacy/ConvertHitObjectParser.cs

```diff
@@ -222,8 +225,12 @@ public override HitObject Parse(string text)
             }
             catch (FormatException)
             {
-                throw new FormatException("One or more hit objects were malformed.");
             }
+            catch (OverflowException)
+            {
+            }
+
+            return null;
```

Comment:
I don't like having no knowledge that the map failed (at least partially) to load.

Reply:
Would you see the user being notified? Or just developer level (ie. logs)? I could see this being a bit annoying if the user is made aware due to aspire maps containing this kind of broken stuff (although I'm neutral on this).